fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/detail/event/rollcall/RollCallDetailFragment.java

```diff
@@ -27,10 +28,10 @@
   private RollCallFragmentBinding mRollCallFragBinding;
   private LaoDetailViewModel mLaoDetailViewModel;
 
-  public static RollCallDetailFragment newInstance(String pk) {
+  public static RollCallDetailFragment newInstance(PublicKey pk) {
     RollCallDetailFragment rollCallDetailFragment = new RollCallDetailFragment();
     Bundle bundle = new Bundle(1);
-    bundle.putString(EXTRA_PK, pk);
+    bundle.putString(EXTRA_PK, pk.getEncoded());
```

Comment:
How do we know when reading `pk`, if we should read it as public key or as private key ? Is it an implicit convention ? I would largely prefer to read an explicit publicKey that immediately avoids confusion (at least for newbies landing on the code). 
My reference of choice is Uncle Bob's clean code: he states that clean code is as readable as prose and it avoids WTFs (translate: surprises) in the mind of the reviewers ;-)

Reply:
I think pk is used quite a lot in the project. It might be a good idea to create a dedicated PR to addess that.

Reply:
I just checked, and it is in fact used in 8 different places. I stand n my idea to do a separate PR.

Reply:
fair enough :-)